tests/device_tests/bitcoin/test_signmessage.py

```diff
@@ -283,6 +283,11 @@ def test_signmessage(
 @pytest.mark.skip_t1
 @pytest.mark.parametrize("message", MESSAGE_LENGTHS)
 def test_signmessage_pagination(client: Client, message):
+    # Czech characters seem to cause panic in R model
```

Comment:
let's figure out and fix this instead. Rust should be perfectly capable of handling utf characters